app.js

```diff
@@ -21,5 +23,39 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.get('/', (req, res) => {
   res.render('index');
 });
+app.get('/beers/', (req, res) => {
+
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', { beersFromApi })
+    })
+    .catch(error => console.log(error));
+
+});
+
+app.get('/beers/id', (req, res) => {
```

Comment:
```suggestion
app.get('/beers/:id', (req, res) => {
```
Con los dos puntos le indicas que va a recibir algo como parámetro y puedes utilizar la propiedad _params_ para manejarlo, en este caso podríamos guardar en una variable el valor de _req.params.id_ , para luego utilizarlo donde lo necesitáramos, por ejemplo para pasárselo como argumento a un método de una api que utilice el id para algo.

Te dejo [link](https://expressjs.com/es/api.html#req.params) a la documentación de express, donde vienen ejemplos y la explicación es muy clara.